sv_pipeline/genome/utils/mapping_gene_ids_tests.py

```diff
@@ -19,77 +19,86 @@
 class LoadGencodeTestCase(unittest.TestCase):
 
     @mock.patch('sv_pipeline.genome.utils.mapping_gene_ids.logger')
+    @mock.patch('sv_pipeline.genome.utils.mapping_gene_ids.path_exists')
     @mock.patch('sv_pipeline.genome.utils.mapping_gene_ids.pickle')
-    @mock.patch('sv_pipeline.genome.utils.mapping_gene_ids.os.path.isfile')
     @mock.patch('sv_pipeline.genome.utils.mapping_gene_ids.open')
-    @mock.patch('sv_pipeline.genome.utils.mapping_gene_ids.download_file')
-    def _test_load_parsed_data_or_download(self, mock_download, mock_open, mock_isfile, mock_pickle, mock_logger):
-        # load from saved pickle
-        mock_isfile.return_value = True
+    @mock.patch('sv_pipeline.genome.utils.mapping_gene_ids.gzip.open')
+    @mock.patch('sv_pipeline.genome.utils.download_utils.open')
+    def test_load_gencode(self, mock_download_open, mock_gopen, mock_open, mock_pickle, mock_path_exists, mock_logger):
```

Comment:
I'm concerned that this is changing - there is nothing in the new file that should require a change to how we load gencode. If you wanted to abstract out some test utilities, can you do that as a separate branch, and then branch this branch off of it? Its really important to me that through the tests we can see that there are no unintended changes from your code change, which is impossible to do right now

Reply:
So, should I create a test update PR first?

Reply:
yes